frontend/javascripts/admin/admin_rest_api.ts

```diff
@@ -958,6 +958,38 @@ export function getNewestVersionForTracing(
   );
 }
 
+export function getSegmentVolumes(
```

Comment:
I really cant tell whether these numbers make any sense so it would be nice if someone could make sure they do :)

Reply:
Not that I think the backend is incorrect, I just hope the frontend parsing is correct

Reply:
You could test this by zooming in a lot and setting the brush size really tiny, so you can have segment volumes of 1 voxel (or 2, or 3, etc). If these match (in vx), you could also manually check the nm³ conversion

Reply:
and how about the bounding boxes?

Reply:
Same thing I’d say, you could use a 1vx-sized brush to draw a 2d shape like
```
xxx
  x
```
…at a known position, and then check that the volume is 4vx, and that the bbox size is 3×2×1. Of course, this can be extended in 3d as well.